src/osd/OSD.cc

```diff
@@ -3712,14 +3717,14 @@ void OSD::handle_pg_peering_evt(
 {
   if (service.splitting(pgid)) {
     peering_wait_for_split[pgid].push_back(evt);
-    return;
+    return -EEXIST;
```

Comment:
right, we don't send split pg now. are you suggesting that we should print some log message in this case? as in luminous, it's an unreachable branch.